### DISCUSSION TITLE
When will the new release be?

### DISCUSSION THREAD

**Discussion by novagc** [flameshot-org/flameshot#3553] [Q&A] [answered]
When will the new release be?

**Comment by mmahmoudian** [comment, marked as answer]
There are quite a few changes and bug fixes listed to be included in v13:

https://github.com/flameshot-org/flameshot/milestone/7

At the moment we don't have a hard date as there are some open Pull Requests that should be addressed and hopefully merged. Our devs have been mainly focused on their personal life lately, therefore no date is set for the next release. Of course you can always access out nightly builds which is based on the latest merged commits:

https://flameshot.org/docs/installation/development-build/
